app/controllers/ems_network_controller.rb

```diff
@@ -25,7 +25,7 @@ def restful?
   def model_feature_for_action(action)
     case action
     when :edit
-      :ems_network_new
+      :update
```

Comment:
It looks like this method is only used in `app/controllers/mixins/ems_common.rb` for `if respond_to?(:model_feature_for_action) && !@ems.supports?(model_feature_for_action(:edit))`

So once we add `supports :update` to all the EMSs we can just delete these completely and change that line to `ems.supports?(:edit)`

Reply:
Yea. I was tempted to do this but wasn't ready to commit that we have supports setup for the dozen classes that included this mixin.

Reply:
let me try this again.

I wanted to just put `@ems.supports?(:update)`
But that requires that every class the imports EmsCommon (14 classes) to `supports :update`
I wasn't ready to make this commitment.